src/Gui/BitmapFactory.cpp

```diff
@@ -677,3 +673,14 @@ QIcon BitmapFactoryInst::mergePixmap (const QIcon &base, const QPixmap &px, Gui:
 
     return overlayedIcon;
 }
+
+qreal BitmapFactoryInst::getMaximumDPR() {
+    qreal dpr = 1.0f;
+
+    QList<QScreen*> screens = QGuiApplication::screens();
+    for (auto screen: screens) {

```

Comment:
You don't need to create that variable: 

```suggestion
    for (auto* screen : QGuiApplication::screens()) {
```